.github/workflows/daily-builds.yml

```diff
@@ -0,0 +1,61 @@
+name: daily-builds
+
+on:
+  # TODO: Remove this. This is for testing
+  pull_request:
+    branches: [main]
```

Comment:
LGTM !

1. please allow a manual trigger as well.
2. You can remove pull_request and bring back the schedule.